src/integrationTest/java/uk/gov/hmcts/reform/sendletter/services/FtpUploadTest.java

```diff
@@ -26,28 +24,11 @@ public void uploads_file() throws Exception {
         FileToSend doc = new FileToSend("hello.zip", "world".getBytes());
         try (LocalSftpServer server = LocalSftpServer.create()) {
             FtpClient client = FtpHelper.getSuccessfulClient(LocalSftpServer.port);
-            client.upload(doc, false, "bulkprint");
+            client.upload(doc, false, LocalSftpServer.SERVICE_FOLDER);
             File[] files = server.lettersFolder.listFiles();
             assertThat(files.length).isEqualTo(1);
             String content = new String(Files.toByteArray(files[0]));
             assertThat(content).isEqualTo("world");
         }
     }
-
-    @Test
```

Comment:
We could also rename the name of this test, as it seems to be the client test. Just a suggestion.

Reply:
you mean the class?

Reply:
sorry, yes, test class